client/components/forms/form-text-input/index.jsx

```diff
@@ -19,25 +18,31 @@ module.exports = React.createClass( {
 		};
 	},
 
-	render: function() {
-		var otherProps = omit( this.props, [ 'className', 'type' ] ),
-			classes = classNames( {
-				'form-text-input': true,
-				'is-error': this.props.isError,
-				'is-valid': this.props.isValid
-			} );
+	focus() {
+		this.refs.textField.focus();
+	},
+
+	render() {
+		const otherProps = omit( this.props, [ 'className', 'type', 'ref' ] );
+		const { className, type, selectOnFocus } = this.props;
+		const classes = classNames( {
```

Comment:
Could probably include `className` in this `classNames` call to avoid the need to call it a second time below. This is legacy but figured worth pointing out.